netbout-web/src/test/casperjs/inbox-noresults-message.js

```diff
@@ -0,0 +1,32 @@
+/*globals casper:false */
```

Comment:
@mkordas I'm not sure, I see it in every test (it's the first time I use casperJs). As far as I understand, it disables global variables. I would leave it there as to not break something.